docs/sources/administration/configuration.md

```diff
@@ -1427,4 +1427,4 @@ Set this to `true` to have date formats automatically derived from your browser
 
 ### default_timezone
 
-Used as the default time zone for user preferences. Can be either `browser` for the browser local time zone or a time zone name from the IANA Time Zone database, such as `UTC` or `Europe/Amsterdam`.
\ No newline at end of file
+Used as the default time zone for user preferences. Can be either `browser` for the browser local time zone or a time zone name from the IANA Time Zone database, such as `UTC` or `Europe/Amsterdam`.
```

Comment:
What's the change here? Can't tell from the diff.

Reply:
There is no changes. 
The difference can be seen in the format "Display the rich diff", actually we make changes in "Display the source diff"

Reply:
Very strange that the GitHub diff shows these blocks as differing, since when I check locally there is no diff.